_posts/2021-06-25-app.md

```diff
@@ -0,0 +1,103 @@
+
+Um ein sinnvolles Lastenradrouting zur verfügung stellen zu können benötigen wir dein Feedback. Nach vielen Nachforschungen und Tests können wir nur mit echten Streckenverläufen und echten Fahrer:Innen das Routing weiter verfeinern.
+
+Um mitzumachen registriere dich unter: https://play.google.com/apps/testing/de.cargorocket
+
+### Feedback
+
+Uns ist bewusst, dass der aktuelle Stand der App bei weitem noch nicht perfekt ist. Hier gibt es einiges zu verbessern und dazu bist du gefragt.
+Damit wir dein Feedback so schnell wie möglich in die App und das Routing integrieren können wären folgende Punkte wichtig:
+
+* Das Routing ist momentan optimiert für kommerzielle Lastenradlogistik und unterscheidet sich daher teilweise stark von einem profil für Freizeit-/Privatbedarfs-Radverkehr.
+
+* In der App befindet sich unter Einstellungen > Feedback eine Feedbackfunktion, die für kurze Hinweise am besten genutzt wird. Auch während des Routings kann mit der "Stern"-Funktion feedback zum aktuellen Streckenabschnitt gegeben werden. Für komplexeres Feedback sendet uns gerne eine [Mail](mailto:team+playweb@cargorocket.de?subject=%5BAPP-BETA%5D%20Feedback).
+
+* Gibt es Schlatflächen und Flows in der App die dir unintuitiv oder übermäßig komplex vorkommen?
+
+* Sind die vorgeschlagenen Routen aus deiner Sicht sinnvoll und falls nein, welche Kriterien sind hierbei entscheidend?
+
+* Würdest du eine Routing App für deine Fahrten generell nutzen? Welche Faktoren sind hier für dich wichtig?
+
+## Technische Details
+
+Im folgenden Abschnitt findet sich eine kurze Beschreibung des technischen Aufbaus unserer App. Diese Beschreibung ist an Entwickler gerichtet und daher teilweise sehr technisch.
+
+Um möglichst schnell beide großen mobilen Plattformen (IOS, Android) bedienen zu können setzten wir bei der Technologiewahl auf ein cross-platform Framework. Zur Auswahl standen dabei unter Anderen [NativeScript](https://nativescript.org/), [Cordova](https://cordova.apache.org/), [Flutter](https://flutter.dev/) und [ReactNative](https://reactnative.dev/).
+Ausschlaggebendes Kriterium bei der Entscheidung war für uns die Unterstützung von Mapbox Navigation.
+
+### Die Anfänge
+Während des Hackathons versuchten wir uns mit einer Implementierung in Flutter. Dabei stießen wir aber auf einen schwerwiegenden Bug bei der bereits existierenden Mapbox Navigation Library ([Github Issue flutter_mapbox_navigation](https://github.com/eopeter/flutter_mapbox_navigation/issues/103#issuecomment-823917948)).
+Daher entschieden wir uns kurzfristig auf einen Fork von [mapbox-gl-directions](https://github.com/mapbox/mapbox-gl-directions) zurückzugreifen und vorerst nur die schon oben gezeigt Webvariante unseres Service anzubieten.
+In der Zukunft werden wir auch im Web wieder einen voll funktionsfähigen Prototypen anbieten. Dieser wird auf Vue 3 basieren und hier in einem weiterem Blog Artikel vorgestellt werden.
```

Comment:
Maybe we shouldn't promise this, that we develop this in near future :D

Reply:
why not? (: If needed we can easily finish it in one week. The code can potentially be 100%copied from the app

Reply:
If we overpromis... shame on us, but I would definetly finalize this webapp. It might live longer than the mobile app itself.

Reply:
Alright, than we have some pressure :D